easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2022.11-gfbf-2022.10.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2022.11'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'gfbf', 'version': '2022.10'}
```

Comment:
note: needs to be bumped to `2022b` once #16961 is merged

Reply:
done